source/classes.tex

```diff
@@ -666,11 +666,19 @@
 and conversion function declarations only;
 when declaring another kind of member the \grammarterm{decl-specifier-seq}
 shall contain a \grammarterm{type-specifier} that is not a \grammarterm{cv-qualifier}.
-The
-\grammarterm{member-declarator-list} can be omitted only after a
-\grammarterm{class-specifier} or an \grammarterm{enum-specifier} or in a
-friend declaration\iref{class.friend}. A
-\grammarterm{pure-specifier} shall be used only in the declaration of a
+The \grammarterm{member-declarator-list} can be omitted only
+after
```

Comment:
I think "can be omitted only after" is problematic phrasing -- it's unclear whether it's giving permission, stating a restriction, or both. (Can a *member-declarator-list* be omitted after *any* *elaborated-type-specifier*, or only some of them?) This is made a little worse by this change, because three of the four top-level grammar productions for *elaborated-type-specifier* aren't allowed in this context, per the end of [dcl.type.elab]p1. This might help:

> The \grammarterm{member-declarator-list} shall not be omitted unless
> the declaration is a friend declaration or its sole constituent is
> * ...